### PR TITLE
Add "String Resources" section to Contributing guide

### PR DESCRIPTION
**Pull request by oguzkocer**
This PR adds a section to Contributing guide for the correct usage & placement of `string-array`s in the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,6 +77,21 @@ Here are the libraries currently maintained and subtree'd:
 * Stores
 * Utils
 
+# String Resources
+
+We use `values/strings.xml` file for *ALL* translatable strings including string arrays. Each element in a string array should be defined as separate string resource first and then the string array should be defined with `translatable="false"` flag. Here is a basic example:
```
Comment (maxme) [inline CONTRIBUTING.md]: We should add the why behind that (ie. GlotPress limitation).

**Comment by maxme** [comment]
We should move the last paragraph of the file to this section too:

> To help ease the translation process we ask that you mark alias string resources - as well as other strings where appropriate - as not translatable. For example `<string name="foo" translatable="false">@string/bar</string>'

**Comment by oguzkocer** [comment]
@maxme it should be ready for another pass, thanks!

**Comment by maxme** [comment]
Looks good, thanks! :shipit:

**Pull request merged**